db/models/feedback.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+const Sequelize = require('sequelize')
+const db = require('APP/db')
+
+const Feedback = db.define('feedback', {
```

Comment:
So this was a small adjustment I made to the google doc. We have a spreadsheet with all of our volunteers, but i think the same form offered the option to provide anonymous/ random feedback. I thought it was odd that both data were on the same table so I broke one out here